nautobot/docs/plugins/development.md

```diff
@@ -106,31 +108,34 @@ class AnimalSoundsConfig(PluginConfig):
 config = AnimalSoundsConfig
 ```
 
-Nautobot looks for the `config` variable within a plugin's `__init__.py` to load its configuration. Typically, this will be set to the PluginConfig subclass, but you may wish to dynamically generate a PluginConfig class based on environment variables or other factors.
+Nautobot looks for the `config` variable within a plugin's `__init__.py` to load its configuration. Typically, this will be set to the `PluginConfig` subclass, but you may wish to dynamically generate a `PluginConfig` class based on environment variables or other factors.
 
 #### PluginConfig Attributes
 
+The configurable attributes for a `PluginConfig` are listed below in alphabetical order.
+
 | Name | Description |
 | ---- | ----------- |
-| `name` | Raw plugin name; same as the plugin's source directory |
-| `verbose_name` | Human-friendly name for the plugin |
-| `version` | Current release ([semantic versioning](https://semver.org/) is encouraged) |
-| `description` | Brief description of the plugin's purpose |
 | `author` | Name of plugin's author |
 | `author_email` | Author's public email address |
-| `base_url` | Base path to use for plugin URLs (optional). If not specified, the project's `name` will be used. |
-| `required_settings` | A list of any configuration parameters that **must** be defined by the user |
-| `default_settings` | A dictionary of configuration parameters and their default values |
-| `min_version` | Minimum version of Nautobot with which the plugin is compatible |
-| `max_version` | Maximum version of Nautobot with which the plugin is compatible |
-| `middleware` | A list of middleware classes to append after Nautobot's build-in middleware |
-| `caching_config` | Plugin-specific cache configuration
+| `base_url` | (Optional) Base path to use for plugin URLs. If not specified, the project's `name` will be used. |
+| `caching_config` | Plugin-specific cache configuration |
 | `custom_validators` | The dotted path to the list of custom validator classes (default: `custom_validators.custom_validators`) |
-| `datasource_contents` | The dotted path to the list of datasource (Git, etc.) content types to register (default: `datasources.datasource_contents` |
-| `graphql_types` | The dotted path to the list of GraphQL type classes (default: `graphql.graphql_types` |
-| `jobs` | The dotted path to the list of Job classes (default: `jobs.jobs` |
+| `datasource_contents` | The dotted path to the list of datasource (Git, etc.) content types to register (default: `datasources.datasource_contents`) |
+| `description` | Brief description of the plugin's purpose |
+| `graphql_types` | The dotted path to the list of GraphQL type classes (default: `graphql.graphql_types)` |
+| `default_settings` | A dictionary of configuration parameters and their default values |
```

Comment:
```suggestion
| `default_settings` | A dictionary of configuration parameters and their default values |
| `description` | Brief description of the plugin's purpose |
| `graphql_types` | The dotted path to the list of GraphQL type classes (default: `graphql.graphql_types)` |
```

Reply:
Why am I so bad at this? It's to keep you busy, that's why!